src/bin/pgcopydb/copydb_schema.c

```diff
@@ -827,17 +827,27 @@ copydb_prepare_table_specs_hook(void *ctx, SourceTable *source)
 		 * Make sure we have proper statistics (relpages) about the table
 		 * before compute the CTID ranges for the concurrent table scans.
 		 */
-		char sql[BUFSIZE] = { 0 };
+		if (specs->estimateTableSizes)
+		{
+			log_notice("Skipping running ANALYZE on table %s as we are "
```

Comment:
This notice message may contain too much information. Maybe I should remove references to table size estimates from the message.

I will not change this until I hear what Dimitri thinks about it.

Reply:
I believe it could be a very simple and quick DEBUG message. I'm not sure it adds anything that the end-user should consider. While it's important to show every SQL query that is send, specifically ANALYZE which can have a non-trivial impact to production, here we're just skipping a query, no need to tell much about it.